testpilot/users/models.py

```diff
@@ -36,4 +38,26 @@ class UserProfile(models.Model):
     def natural_key(self):
         return self.user.natural_key()
 
+    def unsubscribe(self):
```

Comment:
This model method could use a test with the Basket API mocked out. For example, [here's how we test code checking whether a user is a vouched Mozillian against a mocked-out mozillians.org API](https://github.com/mozilla/testpilot/blob/c4c41e893120132639347e41620fc84d054670f4/testpilot/users/tests.py#L304)